test/Interop/Cxx/operators/Inputs/member-inline.h

```diff
@@ -18,6 +18,17 @@ struct LoadableIntWrapper {
   }
 };
 
+struct LoadableBoolWrapper {
+  bool value;
+  bool operator!() {
+    return !value;
+  }
+
+  bool operator()() {
+    return value;
+  }
```

Comment:
Why did this get added?

Reply:
Ah, this was a leftover from when I was working on the test initially. I'll remove it as it's unused.